src/test/resources/dataset-bags/medium/metadata/depositor-info/agreements.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<agreements xsi:schemaLocation="http://easy.dans.knaw.nl/schemas/bag/metadata/agreements/ https://easy.dans.knaw.nl/schemas/bag/metadata/agreements/agreements.xsd"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:dcterms="http://purl.org/dc/terms/"
+            xmlns="http://easy.dans.knaw.nl/schemas/bag/metadata/agreements/">
+    <depositAgreement>
+        <signerId>user001</signerId>
```

Comment:
This isn't valid according to the schema. See easy-schema-examples for a correct example.